zonemaster/zonemaster-gui
# Updates copyright statements and some translation (GUI)

## Purpose

1. License file and footer: In the copyright statement a range of years is given. That requires updates, but year is not needed.
2. Licens file: Nothing is said about copyright for contributors.
3. The copyright statement in the footer is not translated.
4. Untranslated or incorrectly translated strings found in the same file as the copyright statement goes.

## Changes

1. Removes year from copyright and give the correct "translation" of Internetstiftelsen (LICENSE).
2. Adds statement about contributor's copyright (LICENSE).
3. Adds translation mechanism of the statement in the footer (footer.component.html)
4. Adds translation data (i18n/*.json)
5. Updated translation in some of i18n/*.json file where translation was missing or incorrect  

## How to test this PR

1. Check that the footer is correctly translated in all languages.
2. Check that Danish and Swedish are correctly translated where changed.
